compiler/ballerina-lang/src/main/java/io/ballerina/projects/DependencyManifest.java

```diff
@@ -35,14 +35,18 @@
 public class DependencyManifest {
 
     private final String dependenciesTomlVersion;
+    // TODO: can have a PackageVersion or String instead. Seems like semver is the best matching class. Need to check
+    private final SemanticVersion distributionVersion;
```

Comment:
can have a PackageVersion or String instead. Seems like semver is the best matching class. Any thoughts?

Reply:
+1 to have SemanticVersion.